package.json

```diff
@@ -28,7 +28,7 @@
     "all-examples": "node ./utils/ExampleRunner/build-all-examples-cli.js --fromRoot",
     "build-all-examples": "node ./utils/ExampleRunner/build-all-examples-cli.js --build --fromRoot",
     "serve-static-examples": "npx serve .static-examples",
-    "build-and-serve-static-examples": "yarn run build-all-examples && yarn run serve-static-examples",
+    "build-and-serve-static-examples": "export NODE_OPTIONS=--max_old_space_size=8096 && yarn run build-all-examples && yarn run serve-static-examples",
```

Comment:
I dont think this works on windows.   This needs to be like the example below (copy/paste from OHIF) and you need cross-env included as a devDependency
cross-env NODE_ENV=development APP_CONFIG=config/aws_static.js yarn run build && gzip -9 -r dist"